card/src/main/java/com/adyen/checkout/card/CardValidationUtils.kt

```diff
@@ -108,6 +115,24 @@ object CardValidationUtils {
         return FieldState(normalizedSecurityCode, validation)
     }
 
+    fun validateKcpBirthDateOrTaxNumber(birthDateOrTaxNumber: String): FieldState<String> {
```

Comment:
The same way we discussed on the CSE class, It think this validation could be in it's own class. The Card logic is gonna get pretty complicated pretty fast and I think it will be better to have some logic separation, since KCP its a specific thing I think it's better not to have it mixed up with the generic Card logic.

tldr: I think we should move this to something like a `KcpValidationUtils` class